HACKERS

```diff
@@ -244,6 +244,13 @@ the line C<__UNDEFINED__> to the C<=provides> section to get all macros
 implemented this way to be imported into this section, so they all get
 documented as being provided.
 
+If you need to define helper macro which is not part of C<Devel::PPPort> API
+and its usage is only for definition of other C<Devel::PPPort> macros then
+use C<D_PPP_> prefix for this macro name (e.g. C<D_PPP_SVPV_NOLEN_LP_ARG>).
+All defined macros are processed by C<Devel::PPPort> and C<Devel::PPPort>
```

Comment:
`C<Devel::PPPort> and C<Devel::PPPort>` ?? typo? it's mentioned twice?

Reply:
I rewritten it.